deps/build.jl

```diff
@@ -78,4 +111,11 @@ copy_config("logo-32x32.png", juliakspec)
 copy_config("logo-64x64.png", juliakspec)
 
 eprintln("Installing julia kernelspec $spec_name")
-run(`$jupyter kernelspec install --replace --user $juliakspec`)
+run(`$jupyter-kernelspec install --replace --user $juliakspec`)
```

Comment:
The command should definitely be `jupyter kernelspec`, but this workaround should be fine while we work out what is going on with jupyter/notebook#448. We have no plans to install entrypoints to hidden locations any time soon.